pvlib/tests/iotools/test_pvgis.py

```diff
@@ -64,6 +65,15 @@
     [1187.2, 129.59, 8.06, 0.97, 0.97, 0.0],
     [3950.1, 423.28, 14.8, 1.89, 0.69, 0.0]]
 
+data_horizon_abq = pd.Series([9.9, 13.0, 14.5, 15.7, 14.9, 15.3,
```

Comment:
After some pondering, I am guessing abq stands for Albuquerque? I'd probably just nix abq in all instances.